argonaut/src/main/scala/io/finch/argonaut/package.scala

```diff
@@ -51,5 +51,5 @@ package object argonaut {
    * @return Create a Finch ''EncodeJson'' from an argonaut ''EncodeJson''
    */
   implicit def encodeArgonaut[A](implicit encode: EncodeJson[A]): EncodeResponse[A] =
-    EncodeResponse("application/json")(encode.encode(_).nospaces)
+    EncodeResponse("application/json")(encode.encode(_).nospaces.getBytes("UTF-8"))
```

Comment:
Maybe we could add a `EncodeResponse.fromString("application/json")(whatever)` to clean this up a bit?